### DISCUSSION TITLE
No recibo invitación a la misión de Backend

### DISCUSSION THREAD

**Discussion by marcoh11** [Launch-X-Latam/MisionFrontEnd#486] [Team Posts]
Una consulta compañeros explores me inscribí en el formulario para que me inviten a la misión de backend, me uni al equipo de github de backend pero aun no recibo la invitación para entrar al grupo de teams. Agradecería que me puedan apoyar. Gracias

**Comment by doguedogue** [comment]
Viene en el correo, checa tu spam

**Comment by pieropr2k** [comment]
a mi me enviaron la invitacion, sin embargo Microsoft Teams me dice que es invalida

**Comment by pachenico-pixel** [comment]
Yo tampoco recibi nada para unirme al grupo de Microsoft Teams, puede que seamos muchos y todavía esten enviando los accesos.

**Comment by frankYanez** [comment]
Alguien porfa me puede decir donde veo las clases grabadas?

**Comment by jriveramdev** [comment]
@frankYanez en teams tienes la sección de equipos para la parte de Misión FrontEnd, ahí se encuentran los links y orden de las grabaciones.

**Comment by JessicaSomar98** [comment]
Yo no encuentro el formulario ni he recibido ningún correo para unirme al grupo de Backend en Teams, alguien sabe qué puedo hacer?

**Comment by alaanlx** [comment]
Revisen el correo con el que se registraron en Launch X, a mi me llego el 5 de nov el enlace para Backend en Teams
